relay-server/src/actors/project.rs

```diff
@@ -136,9 +136,12 @@ pub struct ProjectConfig {
     pub span_attributes: BTreeSet<SpanAttribute>,
     #[serde(skip_serializing_if = "Vec::is_empty")]
     pub metric_conditional_tagging: Vec<TaggingRule>,
-    /// Exposable features enabled for this project
+    /// Exposable features enabled for this project.
     #[serde(skip_serializing_if = "BTreeSet::is_empty")]
     pub features: BTreeSet<Feature>,
+    /// Transaction renaming rules.
+    #[serde(skip_serializing_if = "Vec::is_empty")]
+    pub tx_name_rules: Vec<TransactionNameRule>,
```

Comment:
Where do you think it's better to exclude those rules? 
We still have to check expiration of the rule as close to the place where we want to apply them as possible. Since they can expire any moment, even while being in transit between different relays.

Reply:
I was specific enough --  I was referring to the serialization part. Before checking if the vector is empty, discard expired rules and then check for the vector's emptiness. Especially at the beginning, this won't make much of a difference, since we're setting 90d TTLs.